src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The name of the interface `AreaCalculator` is not following the checklist item which states not to use verbs for class/interface names. A better name would be `AreaCalculable` to indicate that it's an interface providing the capability to calculate an area.